Azure/azure-cli
# [KeyVault] keyvault create: enable soft-delete by default

Resolve #11039: Soft-delete on by default via the Azure CLI

Previously in `keyvault create`, if the user doesn't specify `--enable-soft-delete`, soft-delete will not be turned on - the `properties.enableSoftDelete` attribute in the returned JSON is `null`.

This PR bumps the API version so that now in `keyvault create`, if `--enable-soft-delete` is not specified, the keyvault will be created with soft-delete turned on by default -  the `properties.enableSoftDelete` attribute is `true`.

User will need to explicitly specify `--enable-soft-delete false` to turn off soft-delete.

To test, run 
```
az keyvault create -g {} -n {}
``` 
and check the `properties.enableSoftDelete` attribute to be `true`.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
